<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VEDA</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="idLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="idStatusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pwLabel">
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pwLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="pwStatusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pwCheckLabel">
     <property name="text">
      <string>비밀번호 확인</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pwCheckLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="pwCheckStatusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>이름</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="nameStatusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>뒤로가기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="text">
        <string>회원가입 완료</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
